src/fluree/db/dbfunctions/fns.cljc

```diff
@@ -425,6 +437,19 @@
       (add-stack ?ctx entry)
       res)))
 
+(defn not=
+  {:doc      "Returns true if two (or more) values are not equal."
+   :fdb/spec nil
+   :fdb/cost "9 + count of numbers in <="}
```

Comment:
I don't understand the cost description. Would something like "9 + number of arguments" be more clear?